evaluators/rank/NdcgEvaluator/mainfest.yml

```diff
@@ -0,0 +1,12 @@
+manifest_version: 1
+name: NDCGEvaluator
+kind: evaluator
+type: pod
+description: evaluates normalized discounted cumulative gain for information retrieval.
+author: Jina AI Dev-Team (dev-team@jina.ai)
+url: https://jina.ai
+vendor: Jina AI Limited
+documentation: https://github.com/jina-ai/jina-hub
+version: 0.0.1
+license: apache-2.0
+keywords: [ndcg, normalized discounted cumulative gain, information retrieval evaluation]
```

Comment:
Should we have word `evaluator` in the keywords?

Reply:
yes good point!